test/cli/test-dag.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const repoPath = require('./index').repoPath
+const describeOnlineAndOffline = require('../utils/on-and-off')
+const ipfs = require('../utils/ipfs-exec')(repoPath)
+
+describe('dag', () => {
+  describeOnlineAndOffline(repoPath, () => {
+    it('get', () => {
```

Comment:
Yeah I wanted to write more extensive tests but I couldn't even get them to run bc of some issue with a networking module